File: Assets/VirtualCity/ProtoDefine/RspGetProxyMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class RspGetProxyMessage {
[ProtoMember(1)]
    public int code;
[ProtoMember(2)]
    public string tip;
[ProtoMember(3)]
    public string payInfo;

    public RspGetProxyMessage() {
    }

    public RspGetProxyMessage(int code, string tip, string payInfo) {
        this.code = code;
        this.tip = tip;
        this.payInfo = payInfo;
    }

    public string getPayInfo() {
        return payInfo;
    }

    public void setPayInfo(string payInfo) {
        this.payInfo = payInfo;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public string getTip() {
        return tip;
    }

    public void setTip(string tip) {
        this.tip = tip;
    }


}
}

File: Assets/VirtualCity/ProtoDefine/Java/system/RspGetProxyMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.server.game.system.entity.FatherProxyInfo;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.List;

@MessageMeta(module = Modules.SYSTEM,cmd = SystemDataPool.RSP_GETPORXY)
public class RspGetProxyMessage extends Message {
    @Protobuf(order = 1)
    private int code;
    @Protobuf(order = 2)
    private String tip;
    @Protobuf(order = 3)
    private String payInfo;

    public RspGetProxyMessage() {
    }

    public RspGetProxyMessage(int code, String tip, String payInfo) {
        this.code = code;
        this.tip = tip;
        this.payInfo = payInfo;
    }

    public String getPayInfo() {
        return payInfo;
    }

    public void setPayInfo(String payInfo) {
        this.payInfo = payInfo;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getTip() {
        return tip;
    }

    public void setTip(String tip) {
        this.tip = tip;
    }


}